doc/doxygen/CMakeLists.txt

```diff
@@ -72,12 +72,9 @@ IF(DEAL_II_DOXYGEN_USE_MATHJAX)
   IF(MATHJAX_PATH AND NOT DEAL_II_DOXYGEN_USE_ONLINE_MATHJAX)
     SET(_mathjax_relpath ${MATHJAX_PATH})
     MESSAGE(STATUS "Using local MathJax: " ${MATHJAX_PATH})
-    SET(MATHJAX_RELPATH_CONFIG "MATHJAX_RELPATH = ${MATHJAX_PATH}")
   ELSE()
-    SET(_mathjax_relpath "https://cdn.mathjax.org/mathjax/latest")
+    SET(_mathjax_relpath "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/")
     MESSAGE(STATUS "Using web-based MathJax: " ${_mathjax_relpath})
-    SET(MATHJAX_RELPATH_CONFIG "")
-    MESSAGE(STATUS "Using default web-based MathJax")
```

Comment:
The old logic didn't make sense and the variable ``MATHJAX_RELPATH_CONFIG`` was unused.